include/openauto/Service/AndroidAutoEntity.hpp

```diff
@@ -19,21 +19,17 @@
 #pragma once
 
 #include <boost/asio.hpp>
-#include <f1x/aasdk/Transport/ITransport.hpp>
-#include <f1x/aasdk/Channel/Control/IControlServiceChannel.hpp>
-#include <f1x/aasdk/Channel/Control/IControlServiceChannelEventHandler.hpp>
-#include <f1x/aasdk/Channel/AV/VideoServiceChannel.hpp>
-#include <f1x/openauto/autoapp/Configuration/IConfiguration.hpp>
-#include <f1x/openauto/autoapp/Service/IAndroidAutoEntity.hpp>
-#include <f1x/openauto/autoapp/Service/IService.hpp>
-#include <f1x/openauto/autoapp/Service/IPinger.hpp>
+#include "aasdk/Transport/ITransport.hpp"
+#include "aasdk/Channel/Control/IControlServiceChannel.hpp"
+#include "aasdk/Channel/Control/IControlServiceChannelEventHandler.hpp"
+#include "aasdk/Channel/AV/VideoServiceChannel.hpp"
+#include "../Configuration/IConfiguration.hpp"
```

Comment:
`..` is generally bad form. Is there another way?

Reply:
CLion renamed a couple of these and added relative paths.   Missed this one, thanks.  Will re-scrub.